Documentation/tutorials/edge-bridge-tutorial.md

```diff
@@ -4,24 +4,20 @@
 - [Overview](#overview)
   - [Environment](#environment)
   - [Prerequisites](#prerequisites)
```

Comment:
Can we also include a link to the Edge tutorial as a prerequisite here? Something like
```
The tutorial app for this exercise already includes the Edge extensions. If you want to learn more about this, check out the Edge tutorial -link- 
```
We can link to the folder itself so it does not break in case we rename the files

Reply:
The latest update looks good to me, thanks